test/hipcub/test_hipcub_util_ptx.cpp

```diff
@@ -598,8 +664,11 @@ void warp_id_kernel(unsigned int* output)
 
 TEST(HipcubUtilPtxTests, WarpId)
 {
-    constexpr unsigned int hardware_warp_size = HIPCUB_WARP_THREADS;
-    const size_t block_size = 4 * hardware_warp_size;
+    constexpr unsigned int hardware_warp_size_32 = HIPCUB_WARP_SIZE_32;
```

Comment:
You can use HIPCUB_WARP_SIZE_32 and HIPCUB_WARP_SIZE_64 directly and you can delete
```
hardware_warp_size_32
hardware_warp_size_64
```